src/main/resources/db/migration/V1_2__create_table_narmeste_leder_relasjon.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE NARMESTE_LEDER_RELASJON
+(
+    id                           SERIAL               PRIMARY KEY,
+    uuid                         VARCHAR(50)          NOT NULL UNIQUE,
+    created_at                   timestamptz          NOT NULL,
```

Comment:
I feks isdialogmote bruker vi timestamp og ikke timestampz. Er det en god grunn til å gjøre det anderledes her?

Reply:
Er to grunner:
1. Appen vi konsumerer fra bruker det
2. Vi gikk over til timestamp med tidssone UTC i isprediksjon etter en diskusjon mmed Anders Topper og de andre utviklerne.
Om jeg husker rett så var det et ønske om å bruke timestamps med UTC uten lokal tiddsone/sommer/vintertid, slik at man datamessig forholder seg til et  "objektivt" tidspunkt uavhengig av hvor hendelsen skjedde.
Så om jeg forstå resultatet av diskjonen var det at det ble lettere å forholde seg til tidspunket relativt til hverandre dersom det brukes timestamp med tiddsone UTC.

Noen tanker eller innspill der?

Reply:
Ja, det høres ok ut det. Men da må man være nøye når man skal konvertere til LocalDate/LocalDateTime. Kommer noen flere kommentarer om det.